relayer/pkg/starknet/utils.go

```diff
@@ -0,0 +1,16 @@
+package starknet
+
+import "math/big"
+
+func BigIntPadBytes(a *big.Int, length int) []byte {
```

Comment:
nit: naming

```suggestion
func PadBytesBigInt(a *big.Int, length int) []byte {
```

Reply:
Additionally, please add 1-liner docs to these functions.

P.S. This fn is so small, not sure it requires a helper.